float64  target_position
float32  speed
float32  effort
---
bool     success
---
float64 position  # The current gripper position (in meters)
bool stalled      # True iff the gripper is exerting max effort and not moving
bool reached_goal # True iff the gripper position has reached the commanded setpoint